doc/lsst.meas.deblender/tasks/lsst.meas.deblender.SourceDeblendTask.rst

```diff
@@ -0,0 +1,66 @@
+.. lsst-task-topic:: lsst.meas.deblender.SourceDeblendTask
+
+#################
+SourceDeblendTask
+#################
+
+``SourceDeblendTask`` Splits blended sources into individual sources.
+
+.. _lsst.meas.deblender.SourceDeblendTask-summary:
+
+Processing summary
+==================
+
+.. If the task does not break work down into multiple steps, don't use a list.
+.. Instead, summarize the computation itself in a paragraph or two.
+
+``SourceDeblendTask`` runs this sequence of operations:
+
+#. Create the task, adding necessary fields to the given schema.
+
+#. Get the psf from the provided exposure and then run deblend().
+
+#. This task has no return value; it only modifies the SourceCatalog in-place.
```

Comment:
This isn't really a useful summary of what the task does: the first two bullets don't really tell us anything beyond the fairly obvious, and the last one isn't really an operation at all — it's a statement of fact.

I don't think it's necessary for you to write a long description of how deblending works here. Perhaps just change the above to say something like:

```
This task performs housekeeping work — updating table schemas and extracting the PSF from the input image — then delegates to `lsst.meas.deblender.deblend` to perform deblending.
The task has no return value; the input `~lsst.afw.table.SourceCatalog` is modified in-place.
```